onnxruntime/core/providers/qnn/builder/qnn_node_group/hardsigmoid_mul_fusion.cc

```diff
@@ -0,0 +1,144 @@
+#include "core/providers/qnn/builder/qnn_node_group/hardsigmoid_mul_fusion.h"
```

Comment:
NOTE: This file contains existing code that was refactored and moved to a separate file. It fuses HardSigmoid -> Mul into a QNN HardSwish op.